packages/web/src/components/premium-content-purchase-modal/components/PurchaseContentFormFields.tsx

```diff
@@ -47,6 +51,10 @@ export const PurchaseContentFormFields = ({
   const payExtraAmountPresetValues = usePayExtraPresets(
     StringKeys.PAY_EXTRA_PRESET_CENT_AMOUNTS
   )
+  const coinflowMaximumCents = useRemoteVar(IntKeys.COINFLOW_MAXIMUM_CENTS)
+  const { isEnabled: isCoinflowEnabled } = useFeatureFlag(
+    FeatureFlags.BUY_WITH_COINFLOW
```

Comment:
we should probably also check if loaded

Reply:
is that necessary actually? if it's not loaded, it'll just default to false. if we change the default to true down the line, that loaded state isn't necessary

Reply:
Not waiting for it to load means that it might be impossible to override locally. A minor thing, but probably worth being safe?

Reply:
gotcha, should change a few locs probably too